libraries.txt

```diff
@@ -8,4 +8,13 @@
 # matches, so we *don't* have the full range of requirements.txt supported
 # syntax available.
 
-matplotlib==3.2.2
+matplotlib==3.3.3
+numpy==1.19.3
+pandas==1.1.4
+torch==1.7.0 ; sys_platform != "win32"
```

Comment:
PyTorch is _enormous_ (~1.5G if memory serves). Do we really want to force everyone who develops the simulator to need to download a huge binary, especially as the majority of users aren't going to actually be using it?

If the objection to TensorFlow was size and portability issues, I'm not sure how PyTorch solves that? (It's bigger and similarly has native features which may or not behave the same across platforms)

Is there an alternative mechanism which competitors who want to do machine learning could use, but which doesn't require imposing such limitations on others?

Reply:
@prophile is there a reason we thought PyTorch was better than TensorFlow, I can't remember what we said in the meeting

Reply:
It was assumed pytorch was smaller than tensorflow, and much easier to install.

Reply:
I'm pretty sure that they both ship binary wheels (at least on linux), so it should just be `pip install ...`. PyTorch looks like it's variable in size across platforms, which _may_ suggest that there are some other things needed on other platforms?

Reply:
In meeting last night, we concluded that chances are either PyTorch or Tensorflow are probably massively overkill for this, and something like scikit-learn would likely cover everything needed, and be much more useful. Not to mention the install is significantly smaller, and thus makes development much easier.

So let's remove PyTorch.

Reply:
https://github.com/srobo/competition-simulator/pull/246/commits/c99e323c274c9e055be6374204a581d390cb039c